pod/live/utils.py

```diff
@@ -155,3 +163,49 @@ def send_email(subject, message, from_email, to_email, cc_email, html_message):
 
     if not DEBUG:
         msg.send()
+
+
+def get_event_id_and_broadcaster_id(request):
+    """
+    Extracts the event ID and broadcaster ID from the given HTTP request.
+    Args:
+        request: An HTTP request object.
+    Returns:
+        A tuple containing the event ID
+        and broadcaster ID extracted from the request body.
+    """
+    body_unicode = request.body.decode("utf-8")
+    body_data = json.loads(body_unicode)
+    event_id = body_data.get("idevent", None)
+    broadcaster_id = body_data.get("idbroadcaster", None)
+    return event_id, broadcaster_id
+
+
+def check_exists(dest_dir_name, is_dir, max_attempt=6):
+    """ Checks whether a file or directory exists."""
+    fct = os.path.isdir if is_dir else os.path.exists
+    type = "Dir" if is_dir else "File"
+    attempt_number = 1
+    while not fct(dest_dir_name) and attempt_number <= max_attempt:
+        logger.warning(f"{type} does not exists, attempt number {attempt_number} ")
+
+        if attempt_number == max_attempt:
+            logger.error(f"Impossible to create dir {dest_dir_name}")
+            raise Exception(
+                f"{type}: {dest_dir_name} does not exists and can't be created")

```

Comment:
+ il n'y a pas de création, contrairement au message d'erreur => supprimer create et ajouter le type 
`logger.error(f"Impossible to get {type}: {dest_dir_name}") `
et 
`f"{type}: {dest_dir_name} does not exists")`
+ modifier le nom de la variable `dest_dir_name` par `resource_name` ou qqch dans le style

Reply:
merci, je corrige